main.js

```diff
@@ -22,7 +23,10 @@ addCharacter('grumpy wizards make toxic brew for the evil queen and jack')
 function renderLetter(item, number) {
   let $letter = document.createElement('span')
   $letter.textContent = item.letter
-  if (myArray[number] === myArray[state['currentCharacter']]) {
+  if (myArray[number] === myArray[state['currentCharacter']] && (state['myCharacters'][number]['failures'] > 0)) {
```

Comment:
Are `item` and `myArray[number]` the same thing? And `myArray` could be named better. It's really hard to tell what `myArray[number]` means in the context of a typing tutor game without reading all of the code. See if you can improve your variable names.

Also, it's better to use dot notation than bracket notation when you know the property keys already.